### PR TITLE
Added user email verification

### PR DESCRIPTION
**Pull request by ashayp22**
- After signing up, the user is sent to in-between screen.
- User can then go to their email inbox and verify their account by clicking a link.
- The user can also resend the email for verification.
- After their email is verified, the user can continue to the rest of the app.

### CODE REVIEW

File: lib/src/screens/emailVerification.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    return PlatformScaffold(
+        appBar: PlatformAppBar(
+          backgroundColor: AppColors.secondaryUofIDark,
+          title: Text(
+            "Verify Email",
+            style: TextStyle(color: Colors.white),
+          ),
+        ),
+        iosContentPadding: true,
+        body: Row(
```
Comment (rahulshamkuwar) [inline lib/src/screens/emailVerification.dart]: Why is there a `Row` here if the only child is the `Column`?  If you wanted to center the column try doing this

```suggestion
        body: Center(
              child: Column(
```

**Review by rahulshamkuwar — CHANGES_REQUESTED** [review]
Probably just make that change for the `Row` to `Center`, otherwise looks great!

**Comment by rahulshamkuwar** [comment]
Hmm... why did the test fail?

**Comment by ashayp22** [comment]
I'm going to test on the emulator.

**Comment by rahulshamkuwar** [comment]
Look at the error, you're missing a `)` somewhere.

**Comment by rahulshamkuwar** [comment]
Ohhh I see what happened lol, here one sec.

**Comment by ashayp22** [comment]
lmao column within a column

**Comment by rahulshamkuwar** [comment]
Yeah exactly loool. It was my mistake for adding the `Coumn` in the code suggestion thing

**Review by rahulshamkuwar — APPROVED** [review]
Looks good

**Pull request merged**